### PR TITLE
full names and working tagging for packetstorm script

### PR DESCRIPTION
**Pull request by HoxhaEndri**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
The names of the cves are now full and consistent. The tags should also be consistent now.
Tested with the first 50 pages.

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)? If possible add a screenshot.**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by m-1-k-3** [comment]
Please add your name to the Authors

**Comment by HoxhaEndri** [comment]
> looks as it gets out of sync:
> 
> ```
> [*] Generating list of URLs of packetstorm advisory page 1046
> [+] Found PoC for CVE-2012-1664 in advisory osCmax 2.5.0 Cross Site Scripting SQL Injection / https://packetstormsecurity.com/files/111559/osCmax-2.5.0-Cross-Site-Scripting-SQL-Injection.html
> [+] Found PoC for CVE-2012-1665 in advisory osCmax 2.5.0 Cross Site Scripting SQL Injection / https://packetstormsecurity.com/files/111559/osCmax-2.5.0-Cross-Site-Scripting-SQL-Injection.html
> [+] Found PoC for CVE-2012-1671 in advisory phpPaleo 4.8b156 Local File Inclusion / https://packetstormsecurity.com/files/111545/phpPaleo-4.8b156-Local-File-Inclusion.html
> [+] Found PoC for CVE-2012-1672 in advisory Pluck 4.7 Cross Site Request Forgery / https://packetstormsecurity.com/files/111582/Pluck-4.7-Cross-Site-Request-Forgery.html
> ```
> 
> If I go to site [1046](https://packetstormsecurity.com/files/tags/exploit/page1046) I can see the Pluck 4.7 CSRF entry ... but it has no CVE identifier:
> 

I will look into it

File: helpers/packet_storm_crawler.sh [hunk 1]
```diff
@@ -11,14 +11,17 @@
...
 
 # Description:  Update script for packetstorm PoC/Exploit collection
 
 URL="https://packetstormsecurity.com/files/tags/exploit/page"
 LINKS="packet_storm_links.txt"
 SAVE_PATH="/tmp/packet_storm"
 EMBA_CONFIG_PATH="./config/"
+TAGS_CVES=""
+NO_DUP_LINKS=""
+NUMBER_OF_PAGES=$(lynx -dump "${URL}" | grep -E "Page 1 of " | sed -r 's/.*of ([0-9])\,/\1/')
```
Comment (m-1-k-3) [inline helpers/packet_storm_crawler.sh:24]: should we do a check to ensure we have a number of pages here?
if ! [[ "${NUMBER_OF_PAGES}" -gt 0 ]]; then
  echo "[-] Number of pages could not be extracted ... exit now"
  exit 1
fi

**Review by m-1-k-3 — CHANGES_REQUESTED** [review]
just two notes for review

File: .github/workflows/packet_storm_crawler.yml [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 
 on:
+  workflow_dispatch:
   schedule:
     - cron: '0 0 * * 0' # do it every week
 
 jobs:
   update_packetstorm_db:
-    if: github.repository_owner == 'e-m-b-a'
+    if: github.repository_owner == 'HoxhaEndri'
```
[the remark this answers is not in the document]
Comment (HoxhaEndri) [reply .github/workflows/packet_storm_crawler.yml:10]: ah yes

File: helpers/packet_storm_crawler.sh (hunk 1 above)
Comment (HoxhaEndri) [reply helpers/packet_storm_crawler.sh:24]: We could, but the number of pages is always present

Comment (m-1-k-3) [reply helpers/packet_storm_crawler.sh:24]: Please include such a quick check.

Comment (HoxhaEndri) [reply helpers/packet_storm_crawler.sh:24]: done

File: .github/workflows/packet_storm_crawler.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (m-1-k-3) [reply .github/workflows/packet_storm_crawler.yml:10]: please fix for merge

[the remark this answers is not in the document]
Comment (HoxhaEndri) [reply .github/workflows/packet_storm_crawler.yml]: > please fix for merge

I thought I did it, but I forgot that it gets ignored by git.

**Review by m-1-k-3 — APPROVED** [review]

**Comment by m-1-k-3** [comment]
Thank you for fixing this script!

**Pull request merged**